crates/config/src/btcio.rs

```diff
@@ -0,0 +1,33 @@
+use serde::Deserialize;
+
+#[derive(Debug, Clone, Deserialize)]
+pub struct BtcIOConfig {
+    /// How often to poll btc client
+    pub client_poll_dur_ms: u32,
+    /// How often to invoke the writer
+    pub write_poll_dur_ms: u64,
+    /// How the fees for are determined.
+    // FIXME: This should actually be a part of signer.
+    pub fee_policy: FeePolicy,
+    /// How much amount(in sats) to send to reveal address
+    pub reveal_amount: u64,
+}
```

Comment:
I think it makes sense to split up writer tx assembly settings from the RPC-related config here.

Also, for consistency would make sense to be `Btcio`.

Reply:
I think it is fine here. We can do that when we actually need it.

> Also, for consistency would make sense to be Btcio.

Yes indeed.

Reply:
You could make an argument that we should even be splitting up the reading and writing into different crates anyways.  These fields really have disjoint uses.

Reply:
We should do it now while we're changing things so that we don't have more work for ourselves in the future and our design reflects how it actually works.

Reply:
We will be splitting writer and most likely the signer, which might even exist together in the upcoming main refactor. So, let's not try to anticipate too many things and bother too much on this split here.

Reply:
However it's undeniable that the reader configuration and the writer/signer configuration should be split up.

I'm not approving this PR until you do this.

Reply:
Alright, splitting this into `Reader` and `Writer` for now. Will think about splitting `Writer` and `Signer` later on. Sounds good?